In-house patch to remove unnecessary dependencies from Heat's
requirements files. The specific reasons are as follows:

httplib2	Not applicable

kombu		Not applicable

pycrypto	Not applicable to Solaris (M2Crypto used instead)

qpid-python	Not applicable

--- heat-2014.2.2/heat.egg-info/requires.txt.~1~	2015-02-05 08:30:41.000000000 -0800
+++ heat-2014.2.2/heat.egg-info/requires.txt	2015-02-23 14:53:28.397432548 -0800
@@ -3,10 +3,8 @@ argparse
 Babel>=1.3
 eventlet>=0.15.1,<0.16.0
 greenlet>=0.3.2
-httplib2>=0.7.5
 iso8601>=0.1.9
 keystonemiddleware>=1.0.0
-kombu>=2.5.0
 lxml>=2.3
 netaddr>=0.7.12
 oslo.config>=1.4.0  # Apache-2.0
@@ -16,7 +14,6 @@ oslo.messaging>=1.4.0,!=1.5.0,<1.6.0
 oslo.utils>=1.0.0                       # Apache-2.0
 PasteDeploy>=1.5.0
 posix_ipc
-pycrypto>=2.6
 python-ceilometerclient>=1.0.6
 python-cinderclient>=1.1.0
 python-glanceclient>=0.14.0
@@ -28,7 +25,6 @@ python-saharaclient>=0.7.3
 python-swiftclient>=2.2.0
 python-troveclient>=1.0.4
 PyYAML>=3.1.0
-qpid-python
 requests>=1.2.1,!=2.4.0
 Routes>=1.12.3,!=2.0
 six>=1.7.0
--- heat-2014.2.2/requirements.txt.~1~	2015-02-05 08:27:27.000000000 -0800
+++ heat-2014.2.2/requirements.txt	2015-02-23 14:53:35.541352823 -0800
@@ -7,10 +7,8 @@ argparse
 Babel>=1.3
 eventlet>=0.15.1,<0.16.0
 greenlet>=0.3.2
-httplib2>=0.7.5
 iso8601>=0.1.9
 keystonemiddleware>=1.0.0
-kombu>=2.5.0
 lxml>=2.3
 netaddr>=0.7.12
 oslo.config>=1.4.0  # Apache-2.0
@@ -20,7 +18,6 @@ oslo.messaging>=1.4.0,!=1.5.0,<1.6.0
 oslo.utils>=1.0.0                       # Apache-2.0
 PasteDeploy>=1.5.0
 posix_ipc
-pycrypto>=2.6
 python-ceilometerclient>=1.0.6
 python-cinderclient>=1.1.0
 python-glanceclient>=0.14.0
@@ -32,7 +29,6 @@ python-saharaclient>=0.7.3
 python-swiftclient>=2.2.0
 python-troveclient>=1.0.4
 PyYAML>=3.1.0
-qpid-python
 requests>=1.2.1,!=2.4.0
 Routes>=1.12.3,!=2.0
 six>=1.7.0
